spec/spec_helper_acceptance.rb

```diff
@@ -12,13 +12,6 @@
     # refresh check if cache needs refresh on next yum command
     on host, 'yum clean expire-cache'
   end
-
-  # In Puppet 7 the locale ends up being C.UTF-8 if it isn't passed.
-  # This locale doesn't exist in EL7 and won't be supported either.
-  # At least PostgreSQL runs into this.
-  if host['hypervisor'] == 'docker' && host['platform'] == 'el-7-x86_64'
-    ENV['LANG'] = 'en_US.UTF-8'
-  end
```

Comment:
This bit is now also redundant:
https://github.com/theforeman/puppet-foreman_proxy_content/blob/0be213954a234cceeac1c236824fa612cae9cfaa/.sync.yml#L5-L6=